cmd/runhcs/signalmap.go

```diff
@@ -1,6 +1,6 @@
 package main
 
-var signalMap = map[string]int{
+var signalMapLcow = map[string]int{
```

Comment:
Hmmmm. I'll talk offline, but why are we re-inventing known constants?

Reply:
Spoke offline this is ok.